scripts/update_stm32_package.py

```diff
@@ -87,6 +87,18 @@ def update_cubes():
 else:
     repo_path = Path(os.getenv("HOME")) / "STM32Cube_repo"
 
+if not os.getenv("ZEPHYR_BASE"):
+    raise Exception("ZEPHYR_BASE Not defined")
+
+if not args.noclean:
+    print("Do you want to clean downloaded repo (" + str(repo_path) + ") at the end of updates?")
+    res = input("(Enter y/n) ").lower()
```

Comment:
suggestion: add default option, so that `Enter` will just choose that option

Reply:
@gmarull Thanks for your review.
I prefer to keep an explicit choice rather than a default one to avoid any mistakes.
(also it is most probable that I would chose  `clean ` as default whereas @erwango would prefer no clean as default :smiley:  )